### ISSUE TITLE
Forecast Q4 2019

### ISSUE THREAD

**Issue by sbarre-esit** [BCDevOps/developer-experience#431]
https://trello.com/c/nGlsnDK3/132-forecast-q4-2019

Carry over work from Q3:
- PROD OCP upgrade to 3.11.146 (to match LAB). ITIMS RFC184443, currently scheduled for start on October 26th. Deferred twice (Oct 5, 19) due to priority given to completing PROD Aporeto install first.

New Q4 Maintenance work planned.

OS Patching:
 - LAB OS Patching (ITIMS RFC186289). Tentatively to start week of October 21st, actual start date dependent both on Red Hat announcing specific errata release for docker and then internal pulp resync of the MCS repo.
- PROD OS Patching (ITIMS RFC 186291). Tentatively to start week of November 18th. Later date is meant to align with proposed PROD OCS upgrade so as to reduce the number of impactful sessions we have with the OCS nodes for the end users.

OCS Upgrades:
- LAB OCS upgrade to 3.11.4 (ITIMS RFC86301). Tentatively to start week of November 4th, contingent on confirming actual release of OCS 3.11.4 the week prior (Matt Robson keeping us informed to this end).
- PROD OCS upgrade to 3.11.4 (ITIMS RFC186326). Tentatively to start week of November 25th. 

No OCP upgrades planned this quarter thus far.
Thus far all maintenance activities will have us completed prior to December holiday change freeze), but if needed, we have three more weeks on the calendar to play with regarding rescheduling efforts.
